src/main/features/ccj/routes/confirmation.ts

```diff
@@ -3,25 +3,18 @@ import * as express from 'express'
 import { Paths } from 'ccj/paths'
 import { ErrorHandling } from 'common/errorHandling'
 import Claim from 'claims/models/claim'
-import { Moment } from 'moment'
 import { CountyCourtJudgment } from 'claims/models/countyCourtJudgment'
-import * as config from 'config'
-
-const timeForResponseInDays: number = config.get<number>('claim.countyCourtJudgment.timeForResponseInDays')
```

Comment:
Seems to no longer be in use. Lets remove it.

Reply:
👍